sql/charsetcollation.go

```diff
@@ -1200,10 +708,10 @@ type mysqlCollationRow struct {
 	PadSpace   string
 }
 
-var CollationToMySQLVals = map[Collation]mysqlCollationRow{
-	Collation_binary:             {63, Y, Y, 0, NoPad},
-	Collation_utf8_general_ci:    {33, Y, Y, 1, PadSpace},
-	Collation_utf8mb4_0900_ai_ci: {255, Y, Y, 0, NoPad},
+var CollationToMySQLVals = map[string]mysqlCollationRow{
+	Collation_binary.Name:             {63, Y, Y, 0, NoPad},
+	Collation_utf8_general_ci.Name:    {33, Y, Y, 1, PadSpace},
+	Collation_utf8mb4_0900_ai_ci.Name: {255, Y, Y, 0, NoPad},
```

Comment:
I think this is fine for now, given the size of the change already. I think we want `CollationToMySQLVals` and `characterSetMaxLengths`, etc. to be inlined into the Collation struct

Reply:
Yeah, true.  Not exactly sure what I want to do with the collations structure / interface, but I'll go back and take a second look soon and figure out what to do about case sensitivity in where clauses which is busted atm.